evennia/commands/default/general.py

```diff
@@ -417,6 +417,44 @@ def func(self):
         caller.location.msg_contents(emit_string, exclude=caller, from_obj=caller)
 
 
+class CmdWhisper(COMMAND_DEFAULT_CLASS):
+    """
+    Speak privately as your character to another
+
+    Usage:
+      whisper <player> = <message>
+
+    Talk privately to those in your current location.
+    """
+
+    key = "whisper"
+    locks = "cmd:all()"
+
+    def func(self):
+        """Run the whisper command"""
+
+        caller = self.caller
+
+        if not self.lhs or not self.rhs:
+            caller.msg("Usage: whisper <player> = <message>")
+            return
+           
+        receiver = caller.search(self.lhs,
+                                 nofound_string="'%s' cannot be found." % self.lhs)
+        if caller == receiver:
+            caller.msg("You can't whisper yourself.")
```

Comment:
This looks like a wrong indent. Also should it not be "You can't whisper TO yourself" ?